doc/SONiC_202006_Release_Notes.md

```diff
@@ -0,0 +1,134 @@
+# SONiC 201911 Release Notes
```

Comment:
s/201911/202006/

Reply:
This was already addressed some time back. Hence, no action.